ConvertAnyStringIntoKebabCaseStyle/Program.cs

```diff
@@ -1,12 +1,20 @@
 ﻿using System;
+using System.Text.RegularExpressions;
+using System.Text;
 
 namespace ConvertAnyStringIntoKebabCaseStyle
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.Write("Enter text convert to kebab-case-style: ");
+            var inputText = (Console.ReadLine());
+            string pattern = @"\W";
```

Comment:
please make variables on line 13 and 14 constant, also move them to the beginning of the Main method. Also, why do you name variable for replacement char 'target'? Please, name it meaningfully, for example replacement (as it is named in Regex method).